CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 ## 4.77.0 (Unreleased)
 
+NOTES:
+* vpcaccess: reverted the ability to update the number of instances for resource `google_vpc_access_connector` ([#15313](https://github.com/hashicorp/terraform-provider-google/pull/15313))
```

Comment:
Do we want a description here of the error users would have seen so they know what is fixed?

Reply:
I guess this happens at the end as well, so nevermind